packages/testing/config/jest/api/globalSetup.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path')
+
+const { getPaths } = require('@redwoodjs/internal/dist/paths')
+
+const rwjsPaths = getPaths()
+
+module.exports = async function () {
+  if (process.env.SKIP_DB_PUSH !== '1') {
+    const process = require('process')
+    // Load dotenvs
+    require('dotenv-defaults/config')
+
+    const cacheDirDb = `file:${path.join(__dirname, '.redwood', 'test.db')}`
+    process.env.DATABASE_URL = process.env.TEST_DATABASE_URL || cacheDirDb
+
+    const command =
+      process.env.TEST_DATABASE_STRATEGY === 'reset'
+        ? ['prisma', 'migrate', 'reset', '--force', '--skip-seed']
+        : ['prisma', 'db', 'push', '--force-reset', '--accept-data-loss']
+
+    const execa = require('execa')
+    execa.sync(`yarn rw`, command, {
+      cwd: rwjsPaths.api.base,
+      stdio: 'inherit',
+      shell: true,
+      env: {
+        DATABASE_URL: process.env.DATABASE_URL,
+      },
+    })
+  }
+}
```

Comment:
Yes you're right about this - but really at this point in time, we have no idea whether the tests that are about to run will use the db or not. Checking for `*.scenarios` (using a glob) can be short sighted too, because if the user isn't using scenarios but seeding the db them selves (as one of the startups do) - it might lead to unexpected behaviour. 

Since this is a "one time thing" - I'm not too concerned about the performance hit here!